src/scripts/imagehelper.js

```diff
@@ -0,0 +1,64 @@
+define(["browser"], function (browser) {
+    "use strict";
+
+    function getDeviceIcon(device) {
+        var baseUrl = "img/devices/";
+        switch (device) {
+            case "Opera":
+            case "Opera TV":
+                return baseUrl + "opera.svg";
+            case "Samsung Smart TV":
```

Comment:
Might want to add LG webOS TV too.

Reply:
The icon pack I used didn't have any for those devices, if someone finds a decent icon later I figure we can add them then.